pkg/restapi/restapi_test.go

```diff
@@ -45,3 +52,16 @@ func TestNew_Success(t *testing.T) {
 	require.NotEmpty(t, controller.GetOperations())
 
 }
+
+func setupLevelDB(t testing.TB) (string, func()) {
```

Comment:
same here - it just creates temp path (returns cleanup)

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/79f650938572126d2758278779bf911ef7c090a4..6f3a7d7e19a26a670b5002e34a70ea25c315587e#diff-aa5d1dd00855c1e6cbeffe4a26f9bc09R56) the name.